File: ios/Pets/Models/Sources/Models/Owner.swift
//
//  File.swift
//  
//
//  Created by Dylan Velez on 11/08/21.
//

import Foundation
public struct Owner: Codable {
    let name, phone, email, address: String
    
    public init(name: String,  phone: String, email: String, address: String) {
        self.name = name
        self.phone = phone
        self.email = email
        self.address = address
    }
}

extension Owner: Hashable { }

File: android/Pets-Starter/app/src/main/java/com/pets/models/Owner.kt
package com.pets.models

import com.google.gson.annotations.SerializedName

data class Owner(
    @SerializedName("name")
    val name: String,
    @SerializedName("phone")
    val phone: String,
    @SerializedName("email")
    val email: String,
    @SerializedName("address")
    val address: String)